libfwupdplugin/fu-udev-device.c

```diff
@@ -816,6 +816,13 @@ fu_udev_device_unbind_driver(FuDevice *device, GError **error)
 	g_autoptr(GOutputStream) stream = NULL;
 
 	/* is already unbound */
+	if (priv->udev_device == NULL) {
+		g_set_error_literal(error,
+				    FWUPD_ERROR,
+				    FWUPD_ERROR_NOT_FOUND,
+				    "not yet initialized");
```

Comment:
Don't use "yet" here to better future proof other lifecycle use.

Reply:
valid, fixed.